KnowWhereGraph/kwg-faceted-search
# Staging deployment & Docs

Fixes issues #298 #294 

This PR adds functionality for

1. Generation the cache from production or staging servers
2. Configurable endpoint so that it can be run on production _and_ stage environments
3. Reduced cache sizes and processing algorithms
4. A dockerfile that contains nginx and the faceted search
5. Documentation for all components and methods
6. Restructured Readme to reflect the code documentation and detail deployment options

To Test:
1. Check this branch out
2. Generate the staging cache
3. Run `ng serve` and confirm that the auto-populating facets work
